deploy/docs/Troubleshoot_Collection.md

```diff
@@ -635,3 +637,48 @@ In order to mitigate this, please use [fluentd-output-sumologic] with `use_inter
 Please follow [Split Big Chunks in Fluentd](./Best_Practices.md#split-big-chunks-in-fluentd)
 
 [fluentd-output-sumologic]: https://github.com/SumoLogic/fluentd-output-sumologic
+
+### Multiline log detection doesn't work as expected
+
+In order to detect multiline logs, we relay on regexes. By default we use the following regex to detect multiline:
+
+- `\[?\d{4}-\d{1,2}-\d{1,2}.\d{2}:\d{2}:\d{2}.*`
```

Comment:
At first, it is hard to see what kind of log will be detected by this regex, maybe would be good to add example case